### DISCUSSION TITLE
What is the Optional Security Onion license key?

### DISCUSSION THREAD

**Discussion by UMHB-InfoSec** [Security-Onion-Solutions/securityonion#11946] [Unsupported Versions] [answered]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/80428885/52e664b1-fcea-41f1-bf8a-deff02aaf8a9)

My google-fu has failed me today.

I have searched the docs and the web but I can't seem to find any information about this setting.

Can someone please explain what this is?

I did accept the elastic license during installation but I don't think this is the same license.

**Comment by dougburks** [comment, marked as answer]
We are building some new enterprise features for Security Onion that will require a license key to unlock. We'll share more information in 2024!

**Reply by UMHB-InfoSec** [reply]
Well, count me in as very interested.  Thanks for the info.
